content/en/docs/cert-compat.md

```diff
@@ -13,7 +13,33 @@ If your certificate validates on some of the "Known Compatible" platforms but no
 
 You may want to visit [this particular community forum discussion](https://community.letsencrypt.org/t/which-browsers-and-operating-systems-support-lets-encrypt/) for more information about compatibility.
 
-# Known Compatible
+From **September 29, 2020**, Let’s Encrypt will change default certificate chain from DST Root X3 to ISRG Root X1. if you want your certificate to be trusted by ISRG Root X1 incompatible clients(ex. Android < 7.1.1), you should manually configure intermediate certificate with DST Root X3 cross-signed one.
+
+From **September 29, 2021**, Let’s Encrypt only issue certificate under ISRG Root X1. if you want your certificate to be trusted by ISRG Root X1 incompatible clients(ex. Android < 7.1.1), you should consider other CA.
```

Comment:
Let's remove this line -- it'll probably be much earlier than that, and we want to have more control over how and when the actual date is announced.